x/valset-pref/msg_server_test.go

```diff
@@ -210,20 +303,33 @@ func (suite *KeeperTestSuite) TestDelegateToValidatorSet() {
 }
 
 func (suite *KeeperTestSuite) TestUnDelegateFromValidatorSet() {
+	suite.SetupTest()
+
+	// prepare existing delegations validators
+	valAddrs := suite.SetupMultipleValidators(3)
+
+	// creates a validator preference list to delegate to
+	preferences := suite.PrepareDelegateToValidatorSet()
+
+	amountToFund := sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 100_000_000)} // 100 osmo
```

Comment:
I think I have seen this used a couple times now so you might want to extract it into a helper function

Reply:
yessir!